### PR TITLE
Distribute directly from s3, not the engage node

### PR DESCRIPTION
**Pull request by djcp**

### CODE REVIEW

**Comment by lbjay** [comment]
:+1:

File: templates/default/config.properties.erb [hunk 1]
```diff
@@ -86,10 +86,9 @@ org.opencastproject.download.directory=${org.opencastproject.shared_storage.dir}
 # The base URL for media downloads.
 <% if @cloudfront_url %>
 org.opencastproject.download.url=https://<%= @cloudfront_url %>
-<% elsif @using_asset_server %>
-org.opencastproject.download.url=http://<%= @asset_server_hostname %>/static
 <% else %>
-org.opencastproject.download.url=<%= (@using_ssl_for_engage) ? 'https://' : 'http://' %><%= @engage_hostname %>/static
+# Distribute directly from the s3 bucket.
```
Comment (rute-santos) [inline templates/default/config.properties.erb]: I believe the virtual-hosted-style url is preferred: https://<%= @s3_distribution_bucket_name %>.s3.amazonaws.com

Comment (djcp) [inline templates/default/config.properties.erb]: nice catch! fixed.

**Comment by djcp** [comment]
oops, this should've been issued against develop (though it's the same as master).  Merged!

**Pull request closed without merging**